draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -848,6 +848,152 @@ Content-Type: text/plain
+   is specific to associating authority with whomever controls the origin
+   server listening for TCP connections on the indicated port of whatever
+   host is identified within the authority component. This is a very weak
+   sense of authority because it depends on both client-specific name
+   resolution mechanisms and communication that might not be secured from
+   man-in-the-middle attacks. Nevertheless, it is a sufficient minimum for
+   binding "http" identifiers to an origin server for consistent resolution
+   within a trusted environment.
+</t>
+<t>
+   If the host identifier is provided as an IP address, the origin server is
+   the listener (if any) on the indicated TCP port at that IP address.
+   If host is a registered name, the registered name is an indirect identifier
+   for use with a name resolution service, such as DNS, to find an address for
+   an appropriate origin server.
+</t>
+<t>
+   Access to "http" services is not limited to the authority mechanism.
+   For example, the Alt-Svc header field <xref target="RFC7838"/> allows an
+   origin server to explicitly alias its own namespace to an alternative
+   service at a different origin. Access to "http" identified resources might
+   also be provided by other protocols outside the scope of this document.
+</t>
+<t>
+   The "https" scheme, in contrast, associates authority based on the ability
+   of a server to use the private key associated with a certificate that
+   the client considers to be trustworthy for the identified host.
+   If a server presents a certificate that verifiably applies to the host,
+   along with proof that it controls the private key, then a client will
+   accept a secured connection to that server as being authoritative for all
+   origins with the same host. However, each origin remains distinct (from
+   origins with a different scheme or port) unless explicitly aliased by an
+   origin server verified to be under the same authority.
+</t>
+<t>
+   What the client is relying on for "https" authority is that there exists
+   some chain of trust (possibly configured by exception) that verifies the
+   certificate provided. Hence, the authority is defined only by the host,
+   the port impacts which certificate is chosen (assuming that certificate
+   matches the host identifier), and the client trusts that match according
+   to the secure handshake and certificate validation described in
+   <xref target="https.initiation"/>.
+</t>
+<t>
+   Note that the "https" scheme does not rely on TCP and the port number for
+   establishing authority, even though it provides for use of HTTP over TLS
+   over TCP via that port as the default mechanism for obtaining a
+   certificate that can be used for verifying authority and providing access
+   to the origin's resources. The peers establishing a secure connection
+   cannot trust transport parameters outside the secured channel, but the
```

Comment:
> The protection offered by the secure channel does not typically provide integrity protection for the port number, but the value serves to distinguish between different origin namespaces on the same host.  Though it is potentially subject to modification, the port identifier is passed as part of the fields that identify a target resource.

I wanted to say `Host:` here, but h2 uses `:authority` by preference, so...

Reply:
Right, I have included parts of that in the new description.